wrapper.js

```diff
@@ -74,22 +74,26 @@ var main = function(args) {
 	}
 
 	var doLint = function(filepath) {
+		//override for lint
```

Comment:
Why not just modify the original `transformWarning` function?

If JSHint compatibility is your concern, I reckon it's better to use JSHint's own CLI anyway. (I haven't tested JSHint compatibility in ages - in fact, I tend to use ESLint these days.)